routes/users.js

```diff
@@ -10,65 +10,81 @@ router.use(bypassIfAuthorised)
 
 // keep static route at top
 router.get('/register', (req, res) => {
-    res.render('users/register', locals({firstName: 'placeholder'}))
+    res.render('users/register')
 })
 
 router.get('/login', (req, res) => {
     res.render('users/login')
 })
 
 router.post('/login', async (req, res) => {
-    const { email, password } = req.body;
+    const { email, password } = req.body
     try {
         const dbUser = await User.findAll({
             where: {
-              email: email
+                email: email,
             },
-          });
-          if (!dbUser[0]?.dataValues) {
-            res.render('users/login', locals({ email }, 'Incorrect email or password'))
+        })
+        if (!dbUser[0]?.dataValues) {
+            res.render(
```

Comment:
thought: you could probably throw here and only have this render logic once in the catch block, same below when it doesn't match password